src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js

```diff
@@ -87,6 +87,7 @@ define([
          */
         initialize: function() {
             // extend defaults with options
+            defaults.locale = this.sandbox.sulu.getDefaultContentLocale();
```

Comment:
Wouldn't it be better to pass the `locale` in the object of the second parameter of the `extend`? Then the `defaults` object wouldn't have to take state.